### PR TITLE
[FEAT] Countdown progress bar

### PR DESCRIPTION
**Pull request by 66Leo66**
## Types of Changes

- [x] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description

1. The number on countdown alert box will now decrease (instead of not changing)
2. A progress bar is added inside the countdown alert box.

## How Has This Been Tested?

Deployed on Netlify  and tested. Working correctly.

## Screenshots (if appropriate):
image

### CODE REVIEW

File: src/pages/StreetView.vue [hunk 1]
```diff
@@ -88,7 +88,13 @@
                 prominent
                 icon="mdi-clock-fast"
             >
-                {{ $tc('StreetView.countdownAlert', timeCountdown) }}
+                {{ $tc('StreetView.countdownAlert', remainingTime) }}
+                <v-progress-linear
+                    :active="isVisibleCountdownAlert"
+                    color="primary"
+                    v-model="countdownPercentage"
```
Comment (BilelJegham) [inline src/pages/StreetView.vue:95]:
```suggestion
                    v-model="remainingTime"
                    :buffer-value="timeCountdown"
```

You can remove le computed countdownPercentage with the  `buffer-value` prop
 https://vuetifyjs.com/en/api/v-progress-linear/#props-buffer-value

**Review by BilelJegham — CHANGES_REQUESTED** [review]
Hello, thanks for your contribution. 

Great idea to add a progress bar in the alert.

I think it will be better to have the progress in the footer of the alert. Like this :

image
Source: [Chelsea Officer](https://dribbble.com/ChelseaOfficer) https://dribbble.com/shots/5666788-Sense-web-app-toast-alerts

What do you think?

File: src/pages/StreetView.vue (hunk 1 above)
Comment (66Leo66) [reply src/pages/StreetView.vue:95]: As far as what I've tested, the total value is always `100`, as the image attached below.
image

**Comment by 66Leo66** [comment]
> Hello, thanks for your contribution.
> 
> Great idea to add a progress bar in the alert.
> 
> I think it will be better to have the progress in the footer of the alert. Like this :
> 
> **[image]**
> 
> What do you think?

Looks nice, but the color and height may need some extra tweak ~~which are things I'm not good at~~
image

File: src/pages/StreetView.vue [hunk 2]
```diff
@@ -88,7 +88,14 @@
                 prominent
                 icon="mdi-clock-fast"
             >
-                {{ $tc('StreetView.countdownAlert', timeCountdown) }}
+                {{ $tc('StreetView.countdownAlert', remainingTime) }}
+                <v-progress-linear
+                    :active="isVisibleCountdownAlert"
+                    color="primary"
```
Comment (BilelJegham) [inline src/pages/StreetView.vue:94]:
```suggestion
                    color="white"
```

Comment (BilelJegham) [inline src/pages/StreetView.vue:94]:
```suggestion
                    color="white"
```

**Comment by BilelJegham** [comment]
> > Hello, thanks for your contribution.
> > Great idea to add a progress bar in the alert.
> > I think it will be better to have the progress in the footer of the alert. Like this :
> > **[image]**
> > What do you think?
> 
> Looks nice, but the color and height may need some extra tweak ~which are things I'm not good at~ image

Hello, sorry for the delay, 

I thinks it will be great with white color

image

**Comment by 66Leo66** [comment]
@BilelJegham Hi, could you please review the changes and provide further opinions?

**Comment by BilelJegham** [comment]
> @BilelJegham Hi, could you please review the changes and provide further opinions?

Hello @66Leo66 ,

image
Could you please fix conflict before the review ?

**Review by BilelJegham — APPROVED** [review]

**Comment by BilelJegham** [comment]
Aprouve with be release in v1.10
Thanks for contribution

@all-contributors please add @66Leo66 for code ;)

**Pull request merged**